deployment/stacks/ui.ts

```diff
@@ -33,8 +34,8 @@ export function createUiComponents(props: CreateUiComponentsProps) {
     isDev,
     cloudfrontCertificateArn,
     cloudfrontDomainName,
-    // vpnIpSetArn,
-    // vpnIpv6SetArn,
+    vpnIpSetArn,
+    vpnIpv6SetArn,
```

Comment:
Let's comment these out everywhere in this file to match MFP and SEDS and CARTS

Reply:
Done.